code/controllers/subsystem/turf_fire_subsystem.dm

```diff
@@ -0,0 +1,9 @@
+SUBSYSTEM_DEF(turf_fire)
+	name = "Turf Fire"
+	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
+	wait = 2 SECONDS
+	var/list/fires = list()
+
+/datum/controller/subsystem/turf_fire/fire()
+	for(var/obj/effect/abstract/turf_fire/fire as anything in fires)
+		fire.process()
```

Comment:
This needs statistics.

If you make it a processing subsystem and use the macros, you'll get the statistics for free.

Reply:
it also needs tick checking